client/app/cdn/dedicated/manage/logs/cdn-dedicated-manage-logs.html

```diff
@@ -0,0 +1,26 @@
+<ovh-tail-logs>
```

Comment:
Regarding my first comment about using the latest version, this component as also be renamed to `<tail-logs>`.